Code/Tools/ProjectManager/Source/ProjectUtils.cpp

```diff
@@ -512,6 +517,97 @@ namespace O3DE::ProjectManager
             return ProjectManagerScreen::Invalid;
         }
 
+        AZ::Outcome<QString, QString> ExecuteCommandResultModalDialog(
+            const QString& cmd,
+            const QStringList& arguments,
+            const QProcessEnvironment& processEnv,
+            const QString& title)
+        {
+            QString resultOutput;
+            QProcess execProcess;
+            execProcess.setProcessEnvironment(processEnv);
+            execProcess.setProcessChannelMode(QProcess::MergedChannels);
+
+            QProgressDialog dialog(title, QObject::tr("Cancel"), /*minimum=*/0, /*maximum=*/0);
+            dialog.setMinimumWidth(500);
+            dialog.setAutoClose(false);
+
+            QProgressBar* bar = new QProgressBar(&dialog);
+            bar->setTextVisible(false);
+            bar->setMaximum(0); // infinite
+            dialog.setBar(bar);
+
+            QLabel* progressLabel = new QLabel(&dialog);
+            QVBoxLayout* layout = new QVBoxLayout();
+
+            // pre-fill the field with the title and command
+            const QString text = QString("%1<br>%2 %3<br>").arg(title).arg(cmd).arg(arguments.join(' '));
+
+            // replace the label with a scrollable text edit
+            QTextEdit* detailTextEdit = new QTextEdit(text, &dialog);
+            detailTextEdit->setReadOnly(true);
+            layout->addWidget(detailTextEdit);
+            layout->setMargin(0);
+            progressLabel->setLayout(layout);
+            progressLabel->setMinimumHeight(150);
+            dialog.setLabel(progressLabel);
+
+            auto readCon = QObject::connect(&execProcess, &QProcess::readyReadStandardOutput,
```

Comment:
nit: Would write out connect or connection in full, per Style Guide because Con could also be interpreted as Console